packages/compass-e2e-tests/helpers/commands/shell-eval.ts

```diff
@@ -19,23 +19,34 @@ export async function shellEval(
 
   const numLines = (await getOutputText(browser)).length;
 
-  const command = parse === true ? `JSON.stringify(${str})` : str;
+  const strings = Array.isArray(input) ? input : [input];
 
-  await browser.setCodemirrorEditorValue(Selectors.ShellInputEditor, command);
-  await browser.keys(['Enter']);
+  for (const str of strings) {
+    const command = parse === true ? `JSON.stringify(${str})` : str;
 
-  // wait until more output appears
-  await browser.waitUntil(async () => {
-    const lines = await getOutputText(browser);
-    return (
-      lines.length >
-      /**
-       * input line becomes an output line on enter press, so we are waiting
-       * for two new lines to appear, not just one
-       */
-      numLines + 1
-    );
-  });
+    await browser.setCodemirrorEditorValue(Selectors.ShellInputEditor, command);
+    await browser.keys(['Enter']);
+
+    // wait until more output appears
+    await browser.waitUntil(async () => {
+      const lines = await getOutputText(browser);
+      if (
+        lines.length >
+        /**
+         * input line becomes an output line on enter press, so we are waiting
+         * for two new lines to appear, not just one
+         */
+        numLines + 1
```

Comment:
aah good catch. missed that when adding the loop. probably doesn't matter because the chevron goes away fast enough (in which case the line count check is probably not necessary anymore), but best to move it anyway.